draftlogs/6918_add.md

```diff
@@ -0,0 +1 @@
+- Add a new attribute `zindex` to Cartesian traces. Traces with higher `zindex` values are drawn in front of traces with lower `zindex` values. This feature was anonymously sponsored: thank you to our sponsor!
```

Comment:
Let's clarify here (and perhaps in the attribute description) that's it only `SVG based Cartesian traces` that have this attribute. Not any WebGL traces e.g. `scattergl`.

Reply:
Good point!